ale/drivers/dawn_driver.py

```diff
@@ -33,7 +33,7 @@ def focal2pixel_lines(self):
         pixel_size = spice.gdpool('INS{}_PIXEL_SIZE'.format(self.ikid), 0, 1)[0] * 0.001
         return [0.0, 0.0, 1/pixel_size]
 
-class DawnPDS3Driver(PDS3, DawnSpice, RadialDistortion):
+class DawnPDS3Driver(Pds3Label, DawnSpice, RadialDistortion):
```

Comment:
How about calling this DawnPDSNaifSpice?

Reply:
May want to keep the 3 in PDS3 unless we want to allow the PDS3 driver to read both PDS3 and PDS4.